doc/example/store.ipynb

```diff
@@ -218,25 +310,123 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Just as in CSV, writing the history to HDF5 takes a considerable amount of time. Note that HDF5 in combination with\n",
+    "`MultiProcessEngine()`will automatically create one file with filename `filename_{id}` for each start, put them in a\n",
+    "folder namen `filename` and create one HDF5 file with name `filename`, which combines all those.The `filename`-file is\n",
+    "intended to be also useable as a savefile for the end results. Note however that `filename` will **cease to function**\n",
+    "if you delete the folder."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "source": [
-    "TODO: This is not fully implemented yet (it's on the way ...)."
+    "print(\"History type: \", type(result.optimize_result.list[0].history))\n",
+    "# print(\"Function value trace of best run: \", result.optimize_result.list[0].history.get_fval_trace())\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 2)\n",
+    "visualize.waterfall(result, ax=ax[0])\n",
+    "visualize.optimizer_history(result, ax=ax[1])\n",
+    "fig.set_size_inches((15, 5))"
+   ],
+   "execution_count": 7,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "History type:  <class 'pypesto.objective.history.CsvHistory'>\n"
```

Comment:
This cell is identical to the previous cell? History type in the output is also `CsvHistory`.